tanner/sessions/session_analyzer.py

```diff
@@ -27,18 +33,132 @@ async def analyze(self, session_key, redis_client):
         else:
             result = await self.create_stats(session, redis_client)
             await self.queue.put(result)
-            await self.save_session(redis_client)
+            await self.save_session(redis_client, pg_client)
 
-    async def save_session(self, redis_client):
+    async def save_session(self, redis_client, pg_client):
         while not self.queue.empty():
+            columns = (
```

Comment:
you have a class for schema creation, you can make it more general and fetch the columns from it